src/utils/index.js

```diff
@@ -28,3 +30,155 @@ export function loadScript(scriptName, cb) {
     const s = document.getElementsByTagName('script')[0];
     s.parentNode.insertBefore(script, s.nextSibling);
 }
+
+function isObject(value) {
+    const type = typeof value;
+    return value != null && (type == 'object' || type == 'function');
+}
+
+export function debounce(func, wait, options) {
```

Comment:
instead of copy pasting the implementation, just use the npm package: https://www.npmjs.com/package/lodash.debounce